.goreleaser.yml

```diff
@@ -58,6 +58,14 @@ brews:
     license: MIT
     test: |
       system "#{bin}/s5cmd version"
+      
+nfpms:
+  -
+    vendor: Peak
+    homepage:  https://github.com/peak/s5cmd
```

Comment:
Could you also add a description, first paragraph of the **Overview** section of README is fine IMHO;
```yaml
    description: |
      Parallel S3 and local filesystem execution tool ... etc
```

If you want to test/inpect produced DEB packages you can run;
```bash
 goreleaser release --skip-publish --skip-validate --rm-dist
```

That would create a directory named `dist` with all artifacts.